### PR TITLE
feat: update README

### PR DESCRIPTION
**Pull request by utkarsh-dixit**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,32 +1,60 @@
...
+
+| Framework | Use-Case | GitHub Repo | Features Used |
+| --- | --- | --- | --- |
+| Autogen | Sample  | [GitHub Repo](https://github.com/username/repo) | - |
+| CrewAI | Competitor research agent that is triggered with a slack message, starts researching about the said topic and  finally creates a page on notion and reply back to slack. | [GitHub Repo](https://github.com/username/repo) | ![](https://img.shields.io/badge/triggers-8A2BE2) ![](https://img.shields.io/badge/actions-8A2BE2) ![](https://img.shields.io/badge/auth-8A2BE2)
+| Langchain | Notion - Product Discussion Agent: An agent that creates linear isseus fronm the production discussions in notion. | [GitHub Repo](https://github.com/username/repo) | ![](https://img.shields.io/badge/actions-8A2BE2) ![](https://img.shields.io/badge/auth-8A2BE2)
+| Lyzr | Repo Recommender - Lyzer agent that analyzes your previous starred repos and notifies you a similar repo that you might be interested in. | [GitHub Repo](https://github.com/username/repo) | ![](https://img.shields.io/badge/actions-8A2BE2) ![](https://img.shields.io/badge/auth-8A2BE2)
+| OpenAI | Sample Use-Case | [GitHub Repo](https://github.com/username/repo) | - |
```
Comment (angrybayblade) [inline README.md:52]: @utkarsh-dixit Can you fix the github links here so we can get this merged?

File: ill.png
Comment (angrybayblade) [inline ill.png:1]: Is this required?

File: README.md (hunk 1 above)
Comment (sohamganatra) [reply README.md:52]: 1. https://github.com/joaomdmoura/crewAI
2. https://github.com/langchain-ai/langchain
3. https://github.com/LyzrCore/lyzr

**Comment by angrybayblade** [comment]
Closing in favour of - https://github.com/SamparkAI/composio/pull/59

**Pull request closed without merging**